### ISSUE TITLE
Endless loop in refresh with no actual connections

### ISSUE THREAD

**Issue by Bolodya1997** [networkservicemesh/sdk#809]
### Env description
Packet automation cluster.

### Steps to reproduce
1. Edit [multi forwarder tests](https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/multiforwarder) to use single NSM setup for running multiple times.
2. Run tests multiple times.

### Actual result
After some time NSMgr starts overusing CPU even if no test is running.
```
NAME                            CPU(cores)   MEMORY(bytes)
forwarder-sriov-hm2hc           1m           17Mi
forwarder-sriov-q6scc           5m           26Mi
forwarder-vpp-4x5tl             87m          356Mi
forwarder-vpp-m72gw             81m          340Mi
nsm-registry-6675db8489-qb98c   1m           12Mi
nsmgr-lmb5r                     1m           13Mi
nsmgr-pxfqr                     4163m        43Mi
```
Profiling for 120s shows that 102s NSMgr spends in refresh.
![image](https://user-images.githubusercontent.com/20160689/113812156-ca034580-9797-11eb-82df-4d31d052d3c8.png)
Logs tail is mostly meaningless and just shows the same: [nsmgr.log](https://github.com/networkservicemesh/sdk/files/6269078/nsmgr.log).

### Expected result
Almost zero CPU usage, time mostly spent in `Register`, `trace`.

**Comment by denis-tingaikin** [comment]
@edwarnicke Do you mind if we move this into `To do`?

**Comment by edwarnicke** [comment]
@denis-tingaikin Please do.  Is this still an active issue?

**Comment by Bolodya1997** [comment]
Fixed with #1017.
